### ISSUE TITLE
file-magic / python-magic

### ISSUE THREAD

**Issue by ninSmith** [GOVCERT-LU/eml_parser#13]
Hi,

My context requires me to use python-magic instead of file-magic.
When using eml_parser, I got the ```AttributeError: module 'magic' has no attribute 'open'```.

I noticed that  in ```eml_parser.py```, instead of having:

```
try:
    import magic
except ImportError:
    magic = None
    magic_mime = None
    magic_none = None
else:
    # MAGIC_MIME_TYPE gives the real mime-type
    magic_mime = magic.open(magic.MAGIC_MIME_TYPE)
    magic_mime.load()
    # MAGIC_NONE gives the meta-information on the analysed file
    magic_none = magic.open(magic.MAGIC_NONE)
    magic_none.load()
```

If I put 

```
import magic
magic = None
magic_mime = None
magic_none = None
```

The parsing works but I don't get the ```mime``` info, which is fine in my use-case.
Do you know a way to handle the wrong magic module (python-magic), in other words parsing the eml even with python-magic and not file-magic ?

Thanks in advance.

**Comment by sim0nx** [comment]
Hi,

I have just uploaded a new version to pypi which includes a workaround for when python-magic is installed instead of file-magic.
It just emits a warning and disables *magic usage.

Hope this helps! :-)

Fixed by 2d74b137cc792d302e1f29d8d94375673d76f1e8.

**Comment by ninSmith** [comment]
It's perfect !

Thanks !
